morpheus/pipeline/linear_pipeline.py

```diff
@@ -113,23 +113,23 @@ def add_segment_boundary(self, data_type=None, as_shared_pointer=False):
         as_shared_pointer : `boolean`
             Whether the data type will be wrapped in a shared pointer.
 
-        Example
-        -------
-            # Create a config and pipeline
-            config = Config()
-            pipe = LinearPipeline(config)
-
-            # Add a source in Segment #1
-            pipe.set_source(FileSourceStage(config, filename=val_file_name, iterative=False))
-
-            # Add a segment boundary
-            # [Current Segment] - [Egress Boundary] ---- [Ingress Boundary] - [Next Segment]
-            pipe.add_segment_boundary(MessageMeta)
-
-            # Add a sink in Segment #2
-            pipe.add_stage(WriteToFileStage(config, filename=out_file, overwrite=False))
-
-            pipe.run()
+        Examples
+        --------
+        >>> # Create a config and pipeline
+        >>> config = Config()
+        >>> pipe = LinearPipeline(config)
+        >>>
+        >>> # Add a source in Segment #1
+        >>> pipe.set_source(FileSourceStage(config, filename=val_file_name, iterative=False))
+        >>>
+        >>> # Add a segment boundary
+        >>> # [Current Segment] - [Egress Boundary] ---- [Ingress Boundary] - [Next Segment]
+        >>> pipe.add_segment_boundary(MessageMeta)
+        >>>
+        >>> # Add a sink in Segment #2
+        >>> pipe.add_stage(WriteToFileStage(config, filename=out_file, overwrite=False))
+        >>>
+        >>> pipe.run()
```

Comment:
Unfortunately Sphinx wants the section to be named "Examples" (not singular) and the code snippets need to have the interpreters ">>>" prompts.

Reply:
I think this is fine. Just need to be consistent moving forwards